main.py

```diff
@@ -1,10 +1,41 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_json_file = json.load(players_file)
+
+    for player, values in players_json_file.items():
+        race, _ = Race.objects.get_or_create(
+            name=values["race"]["name"],
+            description=values["race"]["description"]
+        )
+
+        guild = None
+        guild_info = values.get("guild")
+        if guild_info:
```

Comment:
You could write this condition in one line using walrus operator
```suggestion
        if guild_info := values.get("guild"):
```